skyportal/facility_apis/lco.py

```diff
@@ -539,6 +542,8 @@ def download_observations(request_id, ar):
         session.commit()
     except Exception as e:
         return log(f"Unable to post data for {request_id}: {e}")
+    finally:
+        session.close()
```

Comment:
Do we know the difference between `session.close()` and `session.remove()`?

Reply:
You close a session = Session() and remove a Session().